pennylane/qchem/convert.py

```diff
@@ -1007,3 +1007,108 @@ def _uccsd_state(ccsd_solver, tol=1e-15):
     dict_fcimatr = {key: value for key, value in dict_fcimatr.items() if abs(value) > tol}
 
     return dict_fcimatr
+
+
+def _shci_state(wavefunction, tol=1e-15):
```

Comment:
Is this integrated with `import_state`, or will that come in a later PR?

Reply:
There will be a small PR for that.